### PR TITLE
fix #48931 kaspersky.de

### LINKED ISSUE #48931: kaspersky.de
### Issue URL (Annoyance)
[https://www.kaspersky.de/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.kaspersky.de%2F)
### Comment
Username: @Morku
> Cookie Banner
> --
> Die folgenden Informationen werden automatisch generiert. Bitte löschen Sie sie nicht.
> 
> Custom userscripts:
> https://userscripts.adtidy.org/release/popup-blocker/2.5/popupblocker.user.js
> https://userscripts.adtidy.org/release/assistant/4.3/assistant.user.js
> https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Win
AdGuard version: | 7.3
Browser: | Firefox
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, Send Do-Not-Track header, Remove X-Client-Data header from HTTP requests, Strip URLs from tracking parameters, Self-destructing third-party cookies (180)
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard German

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#48931

### CODE REVIEW

File: AnnoyancesFilter/sections/widgets.txt [hunk 1]
```diff
@@ -305,6 +305,7 @@ amp.focus.de##amp-embed[type="opinary"]
 ! Chat widgets
 !
 moretvtime.com##.cov
+kaspersky.de###nina-kaspersky
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/widgets.txt]: this is not third-party widget. Not sure if we need to block it.

Comment (Alex-302) [reply AnnoyancesFilter/sections/widgets.txt]: @BlazDT @AdamWr

Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/widgets.txt]: should we block it as popup/floating element? @Alex-302

Comment (BlazDT) [reply AnnoyancesFilter/sections/widgets.txt]: As it is not a third-party chat widget, I think it is better not to block it.

**Review by BlazDT — APPROVED** [review]

File: AnnoyancesFilter/sections/widgets.txt (hunk 1 above)
Comment (zloyden) [reply AnnoyancesFilter/sections/widgets.txt]: It's pretty questionable to block it. @Sergey-Lyapin please remove the rule from pr.

**Review by zloyden — APPROVED** [review]

**Pull request merged**